### ISSUE TITLE
React: Leaving IBM - There is no label for the modal dialog#5060

### ISSUE THREAD

**Issue by andysherman2121** [carbon-design-system/carbon-for-ibm-dotcom#5774]
On JAWS FF/Voiceover iPhone
Issue:
There is no label for the modal dialog

Expected:
Provide the heading present in the dialog as the label by associating the same using aria-labelledby

**Comment by RobertaJHahn** [comment]
Closing.  Dup of (#5774)
